src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/SymbolTable.cs

```diff
@@ -1832,7 +1833,7 @@ private static void AddConversionsForOneType(Type type)
             AggregateSymbol aggregate = ((AggregateType)t).OwningAggregate;
 
             // Now find all the conversions and make them.
-            foreach (MethodInfo conversion in type.GetRuntimeMethods())
+            foreach (MethodInfo conversion in type.GetMethods(EverythingBindingFlags))
```

Comment:
This can be just like it was originally: https://github.com/microsoft/referencesource/blob/7f0c570e24e6f0084b3c5794e1a50de9fbe00ca4/Microsoft.CSharp/Microsoft/CSharp/SymbolTable.cs#L2113

- Let the runtime do the filtering using `BindingFlags.Public | BindingFlags.Static`
- `conversion.IsPublic` && `conversion.IsStatic` will become unnecessary

Reply:
Fixed.